src/eth1spec/rlp.py

```diff
@@ -323,4 +323,4 @@ def decode_length(b: Bytes) -> Tuple[Uint, Uint]:
     elif first_rlp_byte <= 0xBF:
         length_length = first_rlp_byte - 0xB7
         rlp_length = BE_inverse(b[1 : 1 + length_length])
-    return rlp_length, 1 + length_length
+    return rlp_length, Uint(1 + length_length)
```

Comment:
Hm, `mypy` couldn't infer the type here?

Reply:
Yeah I'm getting `"rlp_length" is possibly unbound" and I'm not so sure how to resolve it.

Reply:
(seems like that error I'm getting is coming from pyright fwiw)